src/org/ods/services/OpenShiftService.groovy

```diff
@@ -804,7 +804,7 @@ class OpenShiftService {
      */
     String patch(String resource, Map<String, ?> patch, String path = null, String project = null) {
         if (!resource) {
-            throw new IllegalArgumentException('You must specify the resource to patch')
+            throw new IllegalArgumentException('You must specify the resource to path')
```

Comment:
@s2oBCN  this seems wrong, should be patch, not path

Reply:
It was a draft... now is ready ;)